feat: External navigation nodes as part of extensibility (#3075)
kyma-project/busola: docs/extensibility/resources.md

    - **disableCreate** - when set to `true`, it disables the **Create** button. Defaults to `false`.
    - **disableEdit** - when set to `true`, it disables the **Edit** button. Defaults to `false`.
    - **disableDelete** - when set to `true`, it disables the **Delete** button. Defaults to `false`.

> - **externalNodes** - a optional list of links to external websites.

Reviewer: ```suggestion
- **externalNodes** - an optional list of links to external websites.
```

  - **category** - a category name
  - **scope** - either `namespace` or `cluster`. Defaults to `cluster`.

>   - **icon** - an optional icon

Reviewer 2: We need to provide an information where user can find a list of the available icons

>   - **icon** - an optional icon. You can find the list of available icons [here](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html#/overview)

Reviewer: ```suggestion
  - **icon** - an optional icon. Go to [Icon Explorer](https://sdk.openui5.org/test-resources/sap/m/demokit/iconExplorer/webapp/index.html#/overview) to find a list of the available icons.
```

  - **children** - a list of child nodes containing details about the links
    - **label** - a displayed label

>     - **link** - a link to external website. A [JSONata](jsonata.md) function can be provided.

Reviewer: ```suggestion
    - **link** - a link to an external website. You can provide a [JSONata](jsonata.md) function.
```

  actions:
    disableCreate: true
    disableDelete: true

> externalNodes:
>   - category: My Category
>     icon: course-book
>     children:
>       - label: Example Node Label
>         link: 'https://github.com/kyma-project/busola' & $string($exampleResource().link)

Reviewer 2: ```suggestion
externalNodes:
  - category: My Category
    icon: course-book
    children:
      - label: Example Node Label
        link: 'https://github.com/kyma-project/busola'
  - category: My Second Category
    icon: bar-chart
    scope: namespace
    children:
      - label: Example Node Label
        link: '$string($exampleResource().link)'
```
